polling_stations/apps/data_collection/management/commands/import_st_helens.py

```diff
@@ -3,44 +3,45 @@
+    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019sthel.tsv"
+    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019sthel.tsv"
+    elections = ["parl.2019-12-12"]
     csv_delimiter = "\t"
+    allow_station_point_from_postcode = False
 
     def station_record_to_dict(self, record):
-        if record.polling_place_id == "2694":
-            record = record._replace(polling_place_postcode="WA12 0NH")
+        if record.polling_place_id == "3607":
+            record = record._replace(polling_place_easting="0")
+            record = record._replace(polling_place_northing="0")
+
+        if record.polling_place_id == "3670":
+            record = record._replace(polling_place_easting="350436")
+            record = record._replace(polling_place_northing="394900")
         return super().station_record_to_dict(record)
 
     def address_record_to_dict(self, record):
         rec = super().address_record_to_dict(record)
         uprn = record.property_urn.strip().lstrip("0")
 
-        if uprn in [
-            "39079909",  # WN57TY -> WN57TX : 83 Carr Mill Road, Billinge,Nr. Wigan,
-            "39061437",  # WA128RA -> WA128RP : Alexandra Care Home, Park Road South, Newton-le-Willows,, Merseyside
-            "39000688",  # WA104DL -> WA104DN : 49 Alder Hey Road, Eccleston,St Helens,, Merseyside
-            "39068718",  # L354PZ -> L354PX : 2 The Orchard, Rainhill, Merseyside
-            "39086199",  # WN57PG -> WN57PF : 9A Rainford Road, Billinge,Nr. Wigan,
-            "39076000",  # WA118NU -> WA118NT : Ivy Cottage, Ivy Lane, Rainford,St. Helens,, Merseyside
-            "39088582",  # WA95HQ -> WA95EP : 322A Elephant Lane, St Helens,, Merseyside
-            "39075197",  # WA94HX -> WA94HS : 300 Reginald Road, St Helens,, Merseyside
-            "39002933",  # WA102RS -> WA102RL : 40 Baldwin Street, St Helens,, Merseyside
-            "39175085",  # WA118RD -> WA118RG : Fir Tree Farmhouse, 54 Pimbo Road, Kings Moss, St. Helens, Merseyside
-            "39038634",  # WA104QQ -> WA104QF : Mill Bungalow, Mill Brow, Eccleston,St Helens,, Merseyside
-            "39067031",  # WN57TX -> WN57TY : Tan Yard House Farm, Carr Mill Road, Billinge,Nr. Wigan,
-            "39068717",  # L354PZ -> L354PX : 1 The Orchard, Rainhill, Merseyside
-        ]:
-            rec["accept_suggestion"] = True
+        if uprn == "39086246":
+            return None
+
+        # UPRNs 39089002 through 39089022 look fishy. Not sure if it's council or addressbase.
+        if "Alexander Court" in record.addressline1:
+            rec["uprn"] = ""
+
+        if uprn == "39091880":  # 79 Newlove Avenue
+            rec["postcode"] = "WA104DS"
+        if uprn == "39091918":  # 32 Cunningham Court
+            rec["postcode"] = "WA104GA"
+        if uprn == "39099008":  #
+            rec["postcode"] = "WA95RL"
+        if uprn == "39097964":  # 107A Springfield Road
+            rec["postcode"] = "WA103RR"
```

Comment:
Also, there's definitely a few more of these to spot e.g:

WA102TL
14A Hartford Road

WA102TJ
53 The Boulevard